tests/chainer_tests/test_gradient_check.py

```diff
@@ -257,6 +259,209 @@ def test_differenct_eps_gpu(self):
         self.check_different_eps(cuda.to_gpu(self.x), cuda.to_gpu(self.y))
 
 
+default_eps = 1e-3
+
+
+# `result`: True if `func` is non-differentiable on `x`
+@testing.parameterize(*[
+    {'func': 'zero', 'x': [-100.], 'result': False},
+    {'func': 'zero', 'x': [100.], 'result': False},
+    {'func': 'zero', 'x': [0.], 'result': False},
+    {'func': 'zero', 'x': [default_eps / 10], 'result': False},
+    {'func': 'zero', 'x': numpy.random.normal(size=(3, 2)), 'result': False},
+    {'func': 'zero', 'x': numpy.random.normal(size=()), 'result': False},
```

Comment:
Please change size argument to `size=(1,)` to avoid bad indexing during the tests.

Reply:
We have to test against `size=()` anyway.
The error will be gone after https://github.com/cupy/cupy/pull/749.

Reply:
Okay, thank you for the reference. Then let's leave it as it is.